Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: dev push_image
+
+dev:
+	@if [ -z "${FAIRING_DEV_DOCKER_USERNAME}" ]; then\
+		echo "FAIRING_DEV_DOCKER_USERNAME is unset. Set it to your docker username to use this command"; \
+	else \
+		pip install . ;\
+		docker build -t ${FAIRING_DEV_DOCKER_USERNAME}/fairing:latest -f Dockerfile.dev . ; \
```

Comment:
I have some ideas on how we could get rid of scripts like this, but for now you might want to avoid pushing anything with "latest" and tag with git commit sha.

Reply:
This specific script is to make easier to help developers try their changes without having to go through the pip release process.
You would set `FAIRING_DEV_DOCKER_USERNAME` to your username and this would push a dev version to your dockerhub. This image is then pulled instead of the regular base image.
So latest is needed so that you don't have to go modify this script on every change.